.github/workflows/ci.yml

```diff
@@ -47,12 +47,15 @@ jobs:
 
     - name: Build
       run: |
-        make LOCAL_TARGET=bin/linuxkit-${{matrix.arch}} local-build
+        make GOARCH=$(echo ${{matrix.arch}} | cut -d - -f 1) GOOS=$(echo ${{matrix.arch}} | cut -d - -f 2 | cut -d . -f 1) LOCAL_TARGET=bin/linuxkit-${{matrix.arch}} local-build
```

Comment:
Rather than using `cut` it might be cleaner to instead adjust the matrix to look something like this:

```
matrix:
  target:
    - os: linux
      arch: amd64
    - os: darwin
      arch: amd64
     
     ...
 ```
 
 The we can simply reference `${{ matrix.target.os }}` or `${{ matrix.target.arch }}` in various places.
 
 Alternatively, we could go for:
 
 ```
 matrix:
   os:
     - linux
     - windows
     - darwin
    arch:
      - amd64
      - arm64
      - s390x
    exclude:
      - { "os": "darwin", "arch": "s390x"}
      - { "os": "windows", "arch": "s390x"}
      - { "os": "windows", "arch": "arm64"}
```

Reply:
good suggestion. Will update once I can actually produce binaries